llvmlite/ir/values.py

```diff
@@ -14,13 +14,15 @@
 
 _VALID_CHARS = (frozenset(map(ord, string.ascii_letters)) |
                 frozenset(map(ord, string.digits)) |
-                frozenset('._-$'))
+                frozenset(map(ord, ' !#$%&\'()*+,-./:;<=>?@[]^_`{|}~')))
```

Comment:
Well, that adds a py2/py3 branching, which means more opportunity for bugs. I don't see this as necessary since this code is executed exactly once.